src/Razor/src/rzls/Program.cs

```diff
@@ -77,7 +78,7 @@ public static async Task Main(string[] args)
 
         var languageServerFeatureOptions = new ConfigurableLanguageServerFeatureOptions(args);
 
-        var devKitTelemetryReporter = await TryGetTelemetryReporterAsync(telemetryLevel, sessionId, telemetryExtensionPath).ConfigureAwait(true);
+        using var telemetryContext = await TryGetTelemetryReporterAsync(telemetryLevel, sessionId, telemetryExtensionPath).ConfigureAwait(true);
```

Comment:
I was actually surprised this was allowed. Does `Nullable<T>` have a dispose method?

Reply:
No, but a using statement will handle nullable just fine. [SharpLab](https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBBmABM3AYVwG9NcL88EAWXAWQAoBKU8yjhABlwDcIATrgBmuALy4A4gFMALgDEA9opYBudhQC+G3DoAOAqP1nTcSxQH4pc8y3EA+XDGkB3M8rWZ9h46YDOsgIArgDGsu6KuCC4AJIAIlB+eop+EMAANtI6ZBgcnNR0CUkp0iw6HDl5VbiyABYCim7ObgByirIxYHqZYNIwJgAmAKIAHiHSerJQijCeudXa81qYmkA===)

Reply:
neat!